packages/flutter/lib/src/rendering/object.dart

```diff
@@ -912,6 +914,8 @@ class PipelineOwner {
   /// semantics tree.
   final VoidCallback? onSemanticsOwnerCreated;
 
+  final void Function(SemanticsUpdate)? onSemanticsUpdate;
```

Comment:
per style guide: Use a typedef here.

Reply:
This also needs a doc comment.

Reply:
Also: The PipelineOwner must pass this on to the SemanticsOwner somehow? That code seems to be missing? (surprised that no test is failing.... maybe we can add one?)

Reply:
> The PipelineOwner must pass this on to the SemanticsOwner somehow?

The code for bridging this gap can now be found in packages/flutter/lib/src/rendering/object.dart